.github/ISSUE_TEMPLATE/general.md

```diff
@@ -0,0 +1,13 @@
+## Prerequisites
+
+- First of all, thanks for taking part in the community!
```

Comment:
We already have an issue template. We don't want another one.

Reply:
this was aimed to separate bug reports and general issues but that separation might not be so relevant right now